pom.xml

```diff
@@ -0,0 +1,193 @@
+			<id>dev</id>
+			<properties>
+				<old-api-url>10.208.122.38</old-api-url>
+				<new-api-url>10.208.122.38</new-api-url>
+				<environment>dev</environment>
+			</properties>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+		</profile>
+		<profile>
+			<id>uat</id>
+			<properties>
+				<old-api-url>10.208.122.38</old-api-url>
+				<new-api-url>deviemr.piramalswasthya.org</new-api-url>
+				<environment>uat</environment>
+			</properties>
+		</profile>
+		<profile>
+			<id>local</id>
+			<properties>
+				<old-api-url>10.208.122.38</old-api-url>
+				<new-api-url>localhost</new-api-url>
+				<environment>local</environment>
+			</properties>
+		</profile>
+		<profile>
+			<id>test</id>
+			<properties>
+				<old-api-url>10.208.122.38</old-api-url>
+				<new-api-url>10.208.122.32</new-api-url>
+				<environment>test</environment>
+			</properties>
+		</profile>
+    <profile>
+      <id>ci</id>
+      <properties>
+        <environment>ci</environment>
+      </properties>
+    </profile>
+	</profiles>
+
+	<!-- we could create individual properties similar to variables -->
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<final-name>hwc-scheduler-ui-next-v1.0</final-name>
+		<hostname-dev>10.208.122.32</hostname-dev>
+		<port-dev>9990</port-dev>
+		<hostname-local>localhost</hostname-local>
+		<port-local>9990</port-local>
+		<hostname-test>10.208.122.38</hostname-test>
+		<port-test>9990</port-test>
+		<hostname-uat>deviemr.piramalswasthya.org</hostname-uat>
+		<port-uat>9990</port-uat>
+		<server-id>wildfly-${environment}</server-id>
+		<server-hostname>{hostname-${environment}}</server-hostname>
+		<server-port>{port-${environment}}</server-port>
+	</properties>
+
+	<!-- install the necessary node_modules & compile in prod mode -->
+	<build>
+		<finalName>hwc-scheduler-ui-next-v1.0</finalName>
+		<plugins>
+			<plugin>
+				<artifactId>maven-clean-plugin</artifactId>
+				<version>3.3.2</version>
+				<configuration>
+					<filesets>
+						<fileset>
+							<directory>dist</directory>
+							<!-- if any that needs to be included or excluded -->
+							<!-- using name in both would be a crime - pls avoid! -->
+							<!-- <includes><include>**/*.tmp</include><include>**/*.log</include></includes><excludes><exclude>**/important.log</exclude><exclude>**/another-important.log</exclude></excludes> -->
+							<followSymlinks>false</followSymlinks>
+						</fileset>
+					</filesets>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>3.1.0</version>
+				<executions>
+					<execution>
+						<id>npm-install</id>
+						<phase>compile</phase>
+						<goals>
+							<goal>exec</goal>
+						</goals>
+						<configuration>
+							<executable>npm</executable>
+							<arguments>
+								<argument>install</argument>
+								<argument>--legacy-peer-deps</argument>
```

Comment:
Review this argument

Reply:
Ignoring this comment as per MMU-UI-NEXT Comments